docs/user-guide/composer.md

```diff
@@ -4,14 +4,17 @@
  ###  Creating a New Email in web
 1. Click on the "New Email" button.
 2. A new email composer will appear.
-3. From: By default it will display your default identity. When you click on `Edit` icon (3.1)- a drop-down list of your identities will be shown and you can select one of them. The identities you can use in the `From` field can be edited on the [profile page](profile.md).
-4. Fill in the recipient's email address in the "To" field.
-5. Enter a subject for your email in the "Subject" field.
-6. Adding Recipients
+. From: By default it will display your default identity. When you click on `Edit` icon (3.1)- a drop-down list of your identities will be shown and you can select one of them. 
+3. Fill in the recipient's email address in the "To" field.
+4. Enter a subject for your email in the "Subject" field.
+5. Adding Recipients
```

Comment:
```suggestion
3. From: By default it will display your default identity. When you click on `Edit` icon (3.1)- a drop-down list of your identities will be shown and you can select one of them. The identities you can use in the `From` field can be edited on the [profile page](profile.md).
4. Fill in the recipient's email address in the "To" field.
5. Enter a subject for your email in the "Subject" field.
6. Adding Recipients
```